faunadb-java-dsl/src/main/java/com/faunadb/client/types/FaunaField.java

```diff
@@ -0,0 +1,36 @@
+package com.faunadb.client.types;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>Maps a field/getter or constructor parameter to FaunaDB object property while encoding/decoding an object.</p>
+ * <p>
+ * <pre>{@code
+ * class Car {
+ *     @FaunaField("model")
+ *     private String model;
+ *
+ *     @FaunaField("manufacturer")
+ *     private String manufacturer;
+ * }
+ *
+ * class Product {
+ *     private String description;
+ *     private double price;
+ *
+ *     @FaunaConstructor
+ *     public Product(@FaunaField("Description") String description, @FaunaField("Price") double price) {
```

Comment:
These fields are capitalized, while the ones in in `Car` are not capitalized. We should make those consistent.